addons/sale_pdf_quote_builder/models/sale_order.py

```diff
@@ -48,8 +51,28 @@ def _compute_is_pdf_quote_builder_available(self):
                 or order.order_line.available_product_document_ids
             )
 
+    # === CRUD METHODS === #
+
+    @api.model_create_multi
+    def create(self, vals_list):
+        orders = super().create(vals_list)
+        default_documents = self.env['quotation.document'].search([('add_by_default', '=', True)])
+        for order in orders:
+            order.quotation_document_ids |= default_documents & order.available_product_document_ids
+        return orders
+
     # === ACTION METHODS === #
 
+    def get_selected_quotation_documents(self):
```

Comment:
```suggestion
    def _get_selected_quotation_documents(self):
```
Unless they need to be called by rpc, use private methods ;)